### ISSUE TITLE
Crash, Rendering entity in world, because "this.blockEntityRenderDispatcher" is null

### ISSUE THREAD

**Issue by dandin87** [noobanidus/Lootr#103]
All of Fabric 5 - v1.2.2
Minecraft 1.18.2
lootr-1.18.2-0.3.19.55
sodium-fabric-mc1.18.2-0.4.1+build.15
indium-1.0.3+mc1.18.2

I have two reports of this crash in our modpack. 1 on our servers and 1 in single player. It should be noted these are the only two I know of. Anytime they come across a minecart chest. I have no issues and neither do other players so its an odd one. The player on our server has an Intel(R) UHD Graphics 630, no shaders. The single player user has a GeForce RTX 3060 Laptop GPU and is using resourcepacks/shaders.

java.lang.NullPointerException: Cannot invoke "net.minecraft.client.render.block.entity.BlockEntityRenderDispatcher.renderEntity(net.minecraft.block.entity.BlockEntity, net.minecraft.client.util.math.MatrixStack, net.minecraft.client.render.VertexConsumerProvider, int, int)" because "this.blockEntityRenderDispatcher" is null

Intel(R) UHD Graphics 630
https://aofpaste.com/xebabatowu.yaml

NVIDIA GeForce RTX 3060 Laptop GPU
https://aofpaste.com/nonoyuwucu.yaml

**Comment by So1embum** [comment]
I'm having the same issue on a server. I attached my crash log
[crash-2022-05-20_22.17.04-client.txt](https://github.com/noobanidus/Lootr/files/8747018/crash-2022-05-20_22.17.04-client.txt)
.

**Comment by lierbai** [comment]
```
[18:29:19] [Render thread/ERROR]: Reported exception thrown!
net.minecraft.class_148: Rendering entity in world
	at net.minecraft.class_898.method_3954(class_898.java:166) ~[client-intermediary.jar:?]
	at net.minecraft.class_761.method_22977(class_761.java:1563) ~[client-intermediary.jar:?]
	at net.minecraft.class_761.method_22710(class_761.java:1311) ~[client-intermediary.jar:?]
	at net.minecraft.class_757.method_3188(class_757.java:1034) ~[client-intermediary.jar:?]
	at net.minecraft.class_757.method_3192(class_757.java:813) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1122) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:737) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:236) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:460) [fabric-loader-0.14.5.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.14.5.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.14.5.jar:?]
Caused by: java.lang.NullPointerException: Cannot invoke "net.minecraft.class_824.method_23077(net.minecraft.class_2586, net.minecraft.class_4587, net.minecraft.class_4597, int, int)" because "this.blockEntityRenderDispatcher" is null
	at net.zestyblaze.lootr.client.item.LootrChestItemRenderer.renderByMinecart(LootrChestItemRenderer.java:48) ~[S-orC%5B战利品分离%5Dlootr-1.18.2-0.3.19.55.jar:?]
	at net.zestyblaze.lootr.client.entity.LootrChestCartRenderer.render(LootrChestCartRenderer.java:75) ~[S-orC%5B战利品分离%5Dlootr-1.18.2-0.3.19.55.jar:?]
	at net.zestyblaze.lootr.client.entity.LootrChestCartRenderer.method_3936(LootrChestCartRenderer.java:16) ~[S-orC%5B战利品分离%5Dlootr-1.18.2-0.3.19.55.jar:?]
	at net.minecraft.class_898.method_3954(class_898.java:135) ~[client-intermediary.jar:?]
	... 10 more
```

**Comment by noobanidus** [comment]
This should be fixed in [0.3.19.56](https://www.curseforge.com/minecraft/mc-mods/lootr-fabric/files/3800883) which will hopefully be approved soon.
